Remotion/Web/Development.WebTesting/ControlObject.cs

```diff
@@ -46,10 +46,17 @@ protected ControlObject ([NotNull] ControlObjectContext context)
     /// <summary>
     /// Returns the control's HTML ID.
     /// </summary>
-    /// <exception cref="MissingHtmlException">If the <see cref="Scope"/>'s root DOM element does not have an ID attribute.</exception>
+    /// <exception cref="WebTestException">If the <see cref="Scope"/>'s root DOM element does not have an ID attribute.</exception>
     public string GetHtmlID ()
     {
-      return Scope.Id;
+      try
+      {
+        return Scope.Id;
+      }
+      catch (MissingHtmlException exception)
+      {
+        throw new WebTestException ("The control object does not have an ID attribute.", exception);

```

Comment:
Was this something where we want to check with MK if thats really a WebTestException?

Reply:
> 
> 
> Was this something where we want to check with MK if thats really a WebTestException?

If the ID is missing from the `ControlObject`, this would be an error of the developer of the control. This should not happen when using our controls. We should check with MK.